softdevice_controller/include/sdc_hci_vs.h

```diff
@@ -369,6 +378,18 @@ uint8_t sdc_hci_cmd_vs_zephyr_write_bd_addr(const sdc_hci_cmd_vs_zephyr_write_bd
  */
 uint8_t sdc_hci_cmd_vs_zephyr_read_static_addresses(sdc_hci_cmd_vs_zephyr_read_static_addresses_return_t * p_return);
 
+/** @brief Zephyr Read Chip Temperature.
+ *
+ * This commands reads the controller chip temperature.
+ *
+ * @param[out] p_return Extra return parameters.
+ *
+ * @retval 0 if success.
+ * @return Returns value between 0x01-0xFF in case of error.
+ *         See Vol 2, Part D, Error for a list of error codes and descriptions.
+ */
+uint8_t sdc_hci_cmd_vs_zephyr_read_chip_temp(sdc_hci_cmd_vs_zephyr_read_chip_temp_return_t * p_return);
```

Comment:
looks like checkpatch doesn't like the pointer format, but it's autogenerated so maybe we should update the generator

Reply:
edit: just saw your comment for the compliance